framework/src/wifi_manager/wifi_manager_cb.c

```diff
@@ -111,32 +111,32 @@ void _handle_user_cb(_wifimgr_usr_cb_type_e evt, void *arg)
 		}
 		switch (evt) {
 		case CB_STA_CONNECTED:
-			WM_LOG_VERBOSE("[WM] call sta connect success event\n");
+			WM_LOG_INFO("[WM] call sta connect success event\n");
 			cbk->sta_connected(WIFI_MANAGER_SUCCESS);
 			break;
 		case CB_STA_CONNECT_FAILED:
-			WM_LOG_VERBOSE("[WM] call sta connect fail event\n");
+			WM_LOG_INFO("[WM] call sta connect fail event\n");
 			WIFIADD_ERR_RECORD(ERR_WIFIMGR_CONNECT_FAIL);
 			cbk->sta_connected(WIFI_MANAGER_FAIL);
 			break;
 		case CB_STA_DISCONNECTED:
-			WM_LOG_VERBOSE("[WM] call sta disconnect event\n");
+			WM_LOG_INFO("[WM] call sta disconnect event\n");
 			cbk->sta_disconnected(WIFI_MANAGER_DISCONNECT);
 			break;
 		case CB_STA_RECONNECTED:
-			WM_LOG_VERBOSE("[WM] call sta disconnect event\n");
+			WM_LOG_INFO("[WM] call sta disconnect event\n");
```

Comment:
Is this RECONNECT? or DISCONNECT?
If reconnect, we'd better change the log to reconnect.